.cm/gitstream.cm

```diff
@@ -0,0 +1,81 @@
+# -*- mode: yaml -*-
+manifest:
+  version: 1.0
+
+automations:
+  # Enforces a certain format on PR titles
+  enforce_pr_title:
+    if:
+      - {{ pr.title | match(regex=titlePolicy.titleRegex) | nope }}
+    run:
+      - action: request-changes@v1
+        args:
+          comment: |
+            All PRs must be titled according to our semantic naming policy: `[Area]: <short summary>`
+
+            Area must be one of the following:
+             * DataGrid
+             * Pickers 
+             * Charts
+             * TreeView
+             * Chore
+  # Add a label that indicates how many minutes it will take to review the PR.
+  estimated_time_to_review:
+    if:
+      - true
+    run:
+      - action: add-label@v1
+        args:
+          label: "{{ calc.etr }} min review"
+          color: {{ colors.error if (calc.etr >= 20) else (colors.warning if (calc.etr >= 5) else colors.success) }}
+  # Post a comment that lists the best experts for the files that were modified.
+  explain_code_experts:
+    if:
+      - { { pr.labels | match(term='Auto assign') | some } }
+    run:
+      - action: explain-code-experts@v1
+        args:
+          gt: 10
+  # Assign a random reviewer when the 'Share knowledge' label gets added
+  share_knowledge:
+    if:
+      - { { pr.labels | match(term='Share Knowledge') | some } }
+    run:
+      - action: add-reviewers@v1
+        args:
+          reviewers: { { repo | codeExperts(gt=30, lt=60) | random } }
+      - action: add-comment@v1
+        args:
+          comment: |
+            gitStream has assigned a reviewer to increase knowledge sharing on this PR.
+  # Apply a label that indicates when a PR deletes files
+  # This uses the `has` custom expression found at the bottom of this file
+  label_deleted_files:
```

Comment:
it does not happen too often that files are deleted ... this should just raise awareness that this should be taken into account while reviewing.

I am 5/10 that this is really proving to be helpful myself, so i am not against removing it

Reply:
Ok, I'm asking because I can't connect deleting a file to a cases in the past where I wish I didn't miss it. 

I think that the more labels a PR has, the more likely one of its label will be missed, as we don't have the mental capacity to read a large number of labels.